media-gfx/librecad/librecad-2.0.9.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils qmake-utils
+
+DESCRIPTION="Generic 2D CAD program"
+HOMEPAGE="http://www.librecad.org/"
+SRC_URI="https://github.com/LibreCAD/LibreCAD/archive/${PV/_/}.zip -> ${P}.zip"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+IUSE="3d debug doc tools qt4 +qt5"
+REQUIRED_USE="|| ( qt4 qt5 )"
+
+DEPEND="
+	qt4? (
```

Comment:
@jlec please don't suggest to add SLOT operators to Qt dependencies. As they do not explicitly define any subslot right now, the behaviour is undefined and will result in useless rebuilds. The next version of Qt will introduce a subslot but only to handle a specific use case.